cypress/integration/brewery/WorkingInTwoWorkspaces.spec.js

```diff
@@ -235,44 +231,44 @@ describe('Switching between workspaces and running four scenarios at the same ti
+    // create first scenario, update its parameters
     Scenarios.createScenario(sharedNameScenario, true, DATASET.BREWERY_ADT, RUN_TEMPLATE.BREWERY_PARAMETERS).then(
       (value) => {
         firstWorkspaceSharedScenarioId = value.scenarioCreatedId;
-        ScenarioParameters.edit();
+        Scenarios.getScenarioLoadingSpinner(15).should('exist').should('not.be.visible');
         BreweryParameters.getStockInput().clear().type('400');
         BreweryParameters.getRestockInput().clear().type('10');
         BreweryParameters.getWaitersInput().clear().type('15');
-        ScenarioParameters.updateAndLaunch();
+        ScenarioParameters.save();
 
         Workspaces.getHomeButton().should('be.visible').click();
         Workspaces.selectWorkspace(BREWERY_WORKSPACE_ID);
 
-        // create second scenario, update its parameters (same as for the first one) and launch
+        // create second scenario, update its parameters (same as for the first one)
         Scenarios.createScenario(sharedNameScenario, true, DATASET.BREWERY_ADT, RUN_TEMPLATE.BREWERY_PARAMETERS).then(
           (value) => {
             secondWorkspaceSharedScenarioId = value.scenarioCreatedId;
-            ScenarioParameters.edit();
+            Scenarios.getScenarioLoadingSpinner(15).should('exist').should('not.be.visible');
             BreweryParameters.getStockInput().clear().type('400');
             BreweryParameters.getRestockInput().clear().type('10');
             BreweryParameters.getWaitersInput().clear().type('15');
-            ScenarioParameters.updateAndLaunch();
+            ScenarioParameters.save();
 
             // browse to the first scenario and check its parameters
             route.browse({ url: `${REAL_BREWERY_WORKSPACE_ID}/scenario/${firstWorkspaceSharedScenarioId}` });
             // FIXME: find a better work-around to prevent "elements detached from DOM" error. Maybe some queries are
             // causing a re-render of all scenario parameters ?
             // eslint-disable-next-line cypress/no-unnecessary-waiting
             cy.wait(2000);
-            BreweryParameters.getStock().should('have.text', '400');
-            BreweryParameters.getRestock().should('have.text', '10');
-            BreweryParameters.getWaiters().should('have.text', '15');
+            BreweryParameters.getStockInput().should('value', '400');
+            BreweryParameters.getRestockInput().should('value', '10');
+            BreweryParameters.getWaitersInput().should('value', '15');
 
             // browse to the second scenario and check its parameters
             route.browse({ url: `${BREWERY_WORKSPACE_ID}/scenario/${secondWorkspaceSharedScenarioId}` });
-            BreweryParameters.getStock().should('have.text', '400');
-            BreweryParameters.getRestock().should('have.text', '10');
-            BreweryParameters.getWaiters().should('have.text', '15');
+            BreweryParameters.getStockInput().should('value', '400');
```

Comment:
The same, we may consider adding a timeout here: test breaks because cypress tries to check the value before changing the good scenario